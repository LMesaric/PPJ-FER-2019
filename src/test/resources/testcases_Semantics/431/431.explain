semantic: not declared
note: 'l5' on line 19
