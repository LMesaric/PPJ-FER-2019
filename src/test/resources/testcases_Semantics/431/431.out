<primarni_izraz> ::= IDN(19,l5)
